web/components/follow-button.tsx

```diff
@@ -69,3 +73,36 @@ export function UserFollowButton(props: { userId: string; small?: boolean }) {
     />
   )
 }
+
+export function MiniUserFollowButton(props: { userId: string }) {
+  const { userId } = props
+  const currentUser = useUser()
+  const following = useFollows(currentUser?.id)
+  const isFollowing = following?.includes(userId)
+  const user = useUser()
```

Comment:
You're using `useUser` twice. I think the second one should be `useUserById(userId)`

Reply:
Or, you might not need it at all.